homeassistant/components/ebusd/__init__.py

```diff
@@ -95,22 +91,22 @@ def __init__(self, address, circuit):
         self._address = address
         self.value = {}
 
-    @Throttle(MIN_TIME_BETWEEN_UPDATES)
     def update(self, name, stype):
         """Call the Ebusd API to update the data."""
         try:
             _LOGGER.debug("Opening socket to ebusd %s", name)
             command_result = ebusdpy.read(
                 self._address, self._circuit, name, stype, CACHE_TTL
             )
+            _LOGGER.debug("Read result %s", command_result)
             if command_result is not None:
                 if "ERR:" in command_result:
                     _LOGGER.warning(command_result)
                 else:
                     self.value[name] = command_result
         except RuntimeError as err:
             _LOGGER.error(err)
-            raise RuntimeError(err) from err
+            raise RuntimeError(err)
```

Comment:
Why has this been changed? I would recommend to leave it like this.
```suggestion
            raise RuntimeError(err) from err
```